packages/@aws-cdk/aws-rds/test/integ.instance.lit.ts

```diff
@@ -18,19 +18,15 @@ class DatabaseInstanceStack extends cdk.Stack {
     /// !show
     // Set open cursors with parameter group
     const parameterGroup = new rds.ParameterGroup(this, 'ParameterGroup', {
-      engine: rds.DatabaseInstanceEngine.oracleSe1({
-        version: '11.2',
-      }),
+      engine: rds.DatabaseInstanceEngine.ORACLE_SE1,
```

Comment:
We should change to SE2 here also because this integ test will soon not deploy anymore, the instance should also be updated.

Reply:
You're right. I'll do that in a follow-up PR.